umbral/signing.py

```diff
@@ -81,22 +82,22 @@ def __len__(self):
     def __add__(self, other):
```

Comment:
Why `__add__` and `__len__` don't have type hints, but `__bytes__`, `__radd__', and `__eq__` do?

Reply:
other than `__eq__`, They just haven't been covered yet.  As for `__eq__` an implementation change may be needed as `__eq__` needs to be able to take any `object`.